app/infra/eventsService/eventsService.ts

```diff
@@ -143,4 +148,8 @@ ipcRenderer.on(ipcConsts.CLOSING_APP, () => {
   store.dispatch(showClosingAppModal());
 });
 
+ipcRenderer.on(ipcConsts.CHECK_FOR_NEW_UPDATES, (_event, request: { version: string }) => {
```

Comment:
I just noticed.
It seems to me that this will never happen.
What I've missed?

Reply:
It does work. Its triggered when the app is starting.

Reply:
Listens for `CHECK_FOR_NEW_UPDATES` event